src/metabase/email/messages.clj

```diff
@@ -513,25 +513,63 @@
             (when alert-condition-map
               {:alertCondition (get alert-condition-map (pulse->alert-condition-kwd alert))})))))
 
+(defn- schedule-hour-text
+  [{hour :schedule_hour}]
+  (str (if (= hour 0)
```

Comment:
This is fairly simple, but I still wonder if it would be better to use Java date time formatting code to do it?  Like get a `LocalTime` instance from the hour, then format with [`h a`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html)?

Reply:
sure, done